cmd/main.go

```diff
@@ -277,7 +277,7 @@ func main() {
 		// Read the config file
 		file, err := os.Open(config.Configuration.ConfigurationFileName)
 		if err == nil {
-			bs, err := ioutil.ReadAll(file)
+			bs, err := io.ReadAll(file)
 			if err == nil {
 				reader := bytes.NewReader(bs)
 				viper.SetConfigType("toml") // TODO: Set config type based on file extension. Viper supports more than toml.
```

Comment:
Should be able to actually just pass the file into `viper.ReadConfig`. One less check.